ARCHITECTURE.md

```
@@ -0,0 +1,446 @@
+
+3. **Components**
+   - Custom Filament pages for email management
+   - Admin interface for Gmail configuration
+
+## Page Structure
+
+### Laravel Folio Implementation
+
+The site uses Laravel Folio for route-by-blade file structure, allowing direct routing from blade files:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)